src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterTestLocalIT.java

```diff
@@ -116,7 +116,7 @@ void testEmptySelect() throws SQLException {
 	void testSingleLineSelect() throws SQLException {
 		dynamodbTestUtils.createTable(DYNAMO_TABLE_NAME, "isbn");
 		final String ISBN = "12398439493";
```

Comment:
While I know this is an acronym, it is better to write "Isbn" in this case so that it does not look like a static final variable.